### ISSUE TITLE
No honour among thieves

### ISSUE THREAD

**Issue by amylaar** [naev/naev#2097]
I've done various jobs that used to raise my reputation cap with the pirates (e.g. look at the save file from #2091),
and _fcap_pirate is indeed up, but my reputation with all the pirate factions is still limited to 30.

**Comment by bobbens** [comment]
Yes, the pirate faction caps haven't been moved to the new system. This is mainly because I haven't decided on how to deal with the clans. Should they share a cap? Be independent? Be correlated?

**Comment by amylaar** [comment]
Considering each faction keeps track of standing separately, I think each faction should have their own cap, and the location-unspecific missions should have per-faction variables/tags (or array elements).
Maybe a bit of bleed-over of the cap to neighbouring clans?  But not sure if that's worth the complexity.
The Alteris / Delta Pavonis Shady Trader missions are at the fringes of the territories of Wild Ones and Raven Clan, plus there are Galactic Pirates and Marauders present, so I'd say these should be affected.
FLF / Pirate alliance is with Dreamer clan, so that could give some fcap bonus to compensate.
Lotus clan could get some patrol and/or raid bonus.  Or a new Defend the System location for Kretogg / Unicorn that gives an fcap bonus.  But clean no current standing increase for the Empire, like all other dts missions do regardless of faction!.

I wonder who the Proteron DIssidents would sympathize with?  Would it be pirates, or the Empire?

Not sure what to do about the hit_intro.  Make it some famous client / victim so it applies to all pirate factions?
Or attribute it to whichever faction it happens to be with, and compensate with fcap gain from patrols / raids?  Or do the compensation for the non-intro factions by increasing the first-time gains of the hit missions for the factions so each faction gets the same total?  If the latter, it shouldn't be including Highly Dangerous; Dangerous is already very seldom do find and hard to do, I'm not sure if I have seen a Highly Dangerous mission actually.  I suppose to be closest to the spirit to the previous missions, the intro fcap bonus should be given for the first regular mission for a faction if the intro was done for another faction.
That should also be relatively easy to code - it's basically a bonus for the first level zero or higher mission, and the intro mission is the only level zero mission.
